src/useWindowVisibility/useWindowVisibility.ts

```diff
@@ -0,0 +1,24 @@
+import { useCallback, useState } from 'react';
+
+import { useEventListener } from '../useEventListener/useEventListener';
+import { isBrowser } from '../util/const';
+
+/**
+ * Returns a boolean indicating whether the window is visible or not.
+ * @param ssrValue Value to return on server-side rendering.
+ */
+export function useWindowVisibility(ssrValue = true): boolean {
+  const [isVisible, setIsVisible] = useState(document.visibilityState === 'visible');
```

Comment:
This line fails during SSR (as evidenced by the SSR tests failing), because `document` is not available during SSR. You have to check that the hook is being run in a browser before calling browser APIs.

For reference, see the implementation of [useWindowSize](https://github.com/react-hookz/web/blob/master/src/useWindowSize/useWindowSize.ts) (especially lines 34 and 35).